src/Nethermind/Nethermind.Core2/Types/DomainType.cs

```diff
@@ -16,6 +16,9 @@
 
 namespace Nethermind.Core2.Types
 {
+    // TKS: I have changed to enum but now not sure if that was a good choice
+    // I may prefer Sly's solution with a static list as it make the type more in line with everything else
+    // and does not require any special treatment when serializing
```

Comment:
Can not be an enum, as these values are in the config files, so may vary from network to network. They are dynamic values that need to come from Configuration/SignatureDomains

Reply:
ok, will change later (made it enum in another change - here just added a comment for discussion)